stdlib/GZip/src/GZip.jl

```diff
@@ -0,0 +1,480 @@
+## gzip file io ##
```

Comment:
I added new package to stdlib by:

- cd julia/stdlib
- git clone new_repo..
- rename repo from `GZip.jl` to `GZip` (remove `.jl` or any weird chars)
- delete anything superfluous inside GZip, like `.git` files
- add directory name to `stdlib/Makefile`'s `STDLIBS`
- cd to ../../julia
- run make for julia: `make` or debug make `make --debug=j`
- Output should now show new stdlib being built
- Test by opening julia repl, and trying using library, or run its tests `include("test/runtests.jl")`.